packages/design-system-docs/src/pages/components/ChoiceList/checkbox-radio.example.html

```diff
@@ -83,21 +88,20 @@
     <fieldset class="ds-c-fieldset">
       <legend class="ds-c-label"><span>Inverse example</span></legend>
       <span class="ds-c-field__hint ds-c-field__hint--inverse">Helpful hint text</span>
-      <span class="ds-c-field__hint ds-u-color--error-light">Example error message</span>
```

Comment:
I noticed this change, so far we have been using `Example error message` across all the field components. If we make the change here I would prefer to make it everywhere. Otherwise, I would prefer to keep things consistent and leave this out

Reply:
Ah gotcha, I'll revert this then